microsetta_public_api/api/tests/test_api.py

```diff
@@ -1271,6 +1271,52 @@ def test_404_valid(self):
         self.assertEqual(404, response.status_code)
 
 
+class BetaTests(FlaskTests):
+
+    def setUp(self):
+        super().setUp()
+        self.patcher = patch(
+            'microsetta_public_api.api.diversity.beta.k_nearest')
+        self.mock_method = self.patcher.start()
+        _, self.client = self.build_app_test_client()
+
+    def tearDown(self):
+        self.patcher.stop()
+        super().tearDown()
+
+    def test_beta_k_nearest_default_k(self):
```

Comment:
Should there be an API test for a non-existent sample?

Reply:
Sure. I also added an integration test for this. See https://github.com/biocore/microsetta-public-api/pull/69/commits/45a891bb6f9685dc39be3ff4a6d5af20f0e3392b

Reply:
Thanks!